packages/cli-lib/lib/config.js

```diff
@@ -539,6 +543,30 @@ const updateDefaultAccount = defaultAccount => {
   writeConfig();
 };
 
+/**
+ * @throws {Error}
+ */
+const renameAccount = async (currentName, newName) => {
+  const accountId = getAccountId(currentName);
+  const accountConfigToRename = getAccountConfig(accountId);
+  const defaultAccount = getConfigDefaultAccount();
+
+  if (!accountConfigToRename) {
+    throw new Error(`Cannot find account with identifier ${currentName}`);
```

Comment:
I wasn't able to hit this case in my testing. It was updating my account name regardless of what I entered for the currentName.

Reply:
This looks to have been fixed by a recent change.